src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -427,7 +427,11 @@ void MainPage::OnNavPaneOpened(_In_ MUXC::NavigationView ^ sender, _In_ Object ^
 
 void MainPage::OnNavPaneClosed(_In_ MUXC::NavigationView ^ sender, _In_ Object ^ args)
 {
-    KeyboardShortcutManager::HonorShortcuts(true);
+    if (Model->Mode != ViewMode::Graphing)
```

Comment:
Does it make more sense to have a function called KeyboardShortcutManager::DisableShortcuts that does the same thing but overrides whatever value is set by HonorShortcuts? This would make it harder for people to accidentally break this in the future by adding on/off toggles in other parts of the code.